libraries/script-engine/src/UsersScriptingInterface.h

```diff
@@ -112,12 +113,15 @@ public slots:
 
     /**jsdoc
      * Kicks and bans a user. This removes them from the server and prevents them from returning. The ban is by user name if 
-     * available, or machine fingerprint otherwise.
+     * available and by machine fingerprint.
```

Comment:
The ban can be by one or more of user name, machine fingerprint, and IP address.